### ISSUE TITLE
Support of JSON config format ?

### ISSUE THREAD

**Issue by flydev-fr** [mtwebit/DataSet#2]
On my first attempt to use this module, I saw that the YAML PHP extension was required.

What do you think if a choice was available from the DataSet config page to use either YAML or JSON 
 from file field description ?

**Comment by mtwebit** [comment]
JSON is now supported. See https://github.com/mtwebit/DataSet/commit/1634d77b0055c1de0660ba549d4a1b8d59cb325d
There is a problem though: Pagefile uses JSON descriptions internally so it is not so easy to store JSON data there...

**Comment by flydev-fr** [comment]
Thanks @mtwebit 

About the problem, why not storing the JSON in his **own field** (a Textarea ?) instead of the file field description ?

I say that for two reasons, the first for this problem and, the second is that using the description field to store the config require the user to not forget to set the number of rows of the description file field. Not a huge drawback but new user could be confused.

I installed the module with the update but its still complaining about `yaml_parse()` not existing _(I upgraded my MAMP install and not loaded the `YAML extension`)_. It could be better to check if the YAML extension is loaded in first instance and using JSON if the extension is not detected.
